ospf6d/ospf6d.h

```diff
@@ -104,6 +104,12 @@ extern struct thread_master *master;
 		vrf_name = VRF_DEFAULT_NAME;                                   \
 	}
 
+#define OSPF6_FALSE false
+#define OSPF6_TRUE true
```

Comment:
Instead of  using  true /flase  defined these for conventional usage , I have defined these macros. Since these  can be used any where else so defined in common .h file.